examples/advanced/geane/macros/CMakeLists.txt

```
@@ -0,0 +1,30 @@
+ ################################################################################
+ #    Copyright (C) 2019 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+ #                                                                              #
+ #              This software is distributed under the terms of the             # 
+ #              GNU Lesser General Public Licence (LGPL) version 3,             #  
+ #                  copied verbatim in the file "LICENSE"                       #
+ ################################################################################
+GENERATE_ROOT_TEST_SCRIPT(${CMAKE_SOURCE_DIR}/examples/advanced/geane/macros/runMC.C)
```

```suggestion
GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/runMC.C)
```

We can avoid to hard-code the path here. `CMAKE_CURRENT_SOURCE/BINARY_DIR` points to the directory of the closest `CMakeLists.txt`. Apply also in ff.